homework/home1.1/src/Main.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        // Input coefficients
+        System.out.print("Enter coefficient a: ");
```

Comment:
You have copy pasted code for coefficient read for a, b, c - encapsulate it to a separate function and reuse it.